provider/cluster_rosa_classic_resource.go

```diff
@@ -2128,6 +2142,36 @@ func proxyValidators() []tfsdk.AttributeValidator {
 	}
 }
 
+func shouldPatchProperties(state, plan *ClusterRosaClassicState) bool {
+	// User defined properties needs update
+	if _, should := common.ShouldPatchMap(state.Properties, plan.Properties); should {
+		return true
+	}
+
+	extractedDefaults := map[string]string{}
+	for k, v := range state.OCMProperties.Elems {
+		if _, ok := OCMProperties[k]; ok {
+			extractedDefaults[k] = v.(types.String).Value
+		}
+	}
+
+	if len(extractedDefaults) != len(OCMProperties) {
```

Comment:
In case we will delete properties from `OCMProperties` it wouldn't trigger the patch.
Am I right? is it expected?

Reply:
I'm ok not support this

Reply:
You are correct. I do want to support it.
Forced pushed a tweak. Please review.